experimental/list_formatter/examples/and_list.rs

```diff
@@ -23,7 +25,10 @@ fn main(_argc: isize, _argv: *const *const u8) -> isize {
 
     println!(
         "{}",
-        list_formatter.format(&["España", "Francia", "Suiza", "Italia"])
+        list_formatter
+            .format(&["España", "Francia", "Suiza", "Italia"])
+            .to_writeable()
+            .writeable_to_string()
```

Comment:
Note: If we returned `FormattedList` instead of `impl FormattedWriteable`, then we could implement `Writeable` on the concrete type `FormattedList` directly, for example, which could help ergonomics.

Reply:
That would collide with the blanket implementation of `FormattedWriteable` for all `Writeables`. My thinking was that clients should only ever implement one or the other.

Reply:
It shouldn't collide because we have the `to_writeable` function specifically so that we don't have a blanket impl.

Reply:
It collides the other way around. If we implement `Writeable` we cannot implement `FormattedWriteable`.